base/server/src/main/java/org/dogtagpki/server/rest/JobService.java

```diff
@@ -73,12 +107,24 @@ public Response findJobs() throws EBaseException {
         JobsSchedulerConfig jobsSchedulerConfig = engineConfig.getJobsSchedulerConfig();
         JobsConfig jobsConfig = jobsSchedulerConfig.getJobsConfig();
 
+        Principal principal = servletRequest.getUserPrincipal();
+        logger.info("JobService: - principal: " + principal);
+
+        boolean isAdmin = isAdmin(principal);
+
+        logger.info("JobService: - jobs:");
         Enumeration<String> list = jobsConfig.getSubStoreNames().elements();
         while (list.hasMoreElements()) {
             String id = list.nextElement();
-            logger.info("JobService: - " + id);
+            logger.info("JobService:   - " + id);
```

Comment:
Are the added space needed?

Reply:
Yes, it will provide a proper indentation:
```
2022-10-26 09:17:01 [https-jsse-nio-8443-exec-2] INFO: JobService: - jobs:
2022-10-26 09:17:01 [https-jsse-nio-8443-exec-2] INFO: JobService:   - certRenewalNotifier
2022-10-26 09:17:01 [https-jsse-nio-8443-exec-2] INFO: JobService:   - publishCerts
2022-10-26 09:17:01 [https-jsse-nio-8443-exec-2] INFO: JobService:   - requestInQueueNotifier
2022-10-26 09:17:01 [https-jsse-nio-8443-exec-2] INFO: JobService:   - unpublishExpiredCerts
```